apps/web/src/pages/Pools/PoolAdvertisementPage/PoolAdvertisementPage.tsx

```diff
@@ -758,8 +758,8 @@ export const PoolPoster = ({
               <Text>
                 {intl.formatMessage({
                   defaultMessage:
-                    'In order to apply for this job you will need to be able to demonstrate that you have <strong>all the skills</strong> marked <heavyPrimary>"Required"</heavyPrimary>. If you have any of the <heavySecondary>"Optional"</heavySecondary> skills you are encouraged to include them because it will increase your chances of a job placement.',
-                  id: "eihm6A",
+                    "In order to apply for this job you will need to be able to demonstrate that you have <strong>all the skills</strong> marked <heavyPrimary>Required</heavyPrimary>. If you have any of the <heavySecondary>Optional</heavySecondary> skills you are encouraged to include them because it will increase your chances of a job placement.",
```

Comment:
Although I think it makes sense to remove the double quotes around **Required** and **Optional**, my suggestion is not part of the issue's acceptance criteria. You'll need to check with the designer first if this change is approved as this is not in the [design file](https://www.figma.com/file/CdYmWlP6SpT17bw7Q9411Z/Candidates---Job-advertisement?type=design&node-id=439-10852&mode=design&t=i6mUwX1Sa8Dhd0Ei-0) (I don't know whether this is the most up to date file, but it is what I could find).

Reply:
⏪ 
https://github.com/GCTC-NTGC/gc-digital-talent/pull/10077/commits/14fd3a9d47e2ec378c1695c79c554881731a0815